### DISCUSSION TITLE
Document how to calculate 3D geometry in WKT from geometric primitive in cityjson

### DISCUSSION THREAD

**Discussion by Davidswinkels** [cityjson/specs#160] [Ideas]
There is documentation on [geometric primitives](https://www.cityjson.org/dev/geom-arrays/) and [geometry coordinates](https://www.cityjson.org/dev/realcoords/), but it is quite tricky for me as a new user of CityJSON to understand how the geometric coordinates in WKT can be derived from a lod 2.2 feature in 3DBAG. 

Therefore, it would be nice to add an example in https://www.cityjson.org/dev/realcoords/ how to derive 3D geometry in WKT for lod 1 and lod 2.2 from CityJSON.

**Comment by hugoledoux** [comment]
It would be nice, but WKT is not sufficient for all the geometry types of CityJSON (https://www.cityjson.org/specs/1.1.3/#geometry-objects). All the `Solid` and aggregates with those cannot be represented since a `POLYHEDRALSURFACE Z` doesn't allow inner shells (cavities).

So indeed since most buildings and objects in CityJSON do not have inner shells, this could be implemented in the future. 

That's an idea.
